src/main/webapp/app/admin/statistics/statistics.component.ts

```diff
@@ -0,0 +1,178 @@
+import { Component, OnInit, OnChanges, ViewChild } from '@angular/core';
+import { StatisticsService } from 'app/admin/statistics/statistics.service';
+import { SPAN_PATTERN } from 'app/app.constants';
+import { ChartDataSets, ChartOptions, ChartType } from 'chart.js';
+import { BaseChartDirective, Label } from 'ng2-charts';
+import { DataSet } from 'app/exercises/quiz/manage/statistics/quiz-statistic/quiz-statistic.component';
+import { TranslateService } from '@ngx-translate/core';
+import * as moment from 'moment';
+
+export enum SpanType {
+    DAY = 'DAY',
+    WEEK = 'WEEK',
+    MONTH = 'MONTH',
+    YEAR = 'YEAR',
+}
+
+@Component({
+    selector: 'jhi-statistics',
+    templateUrl: './statistics.component.html',
+})
+export class JhiStatisticsComponent implements OnInit, OnChanges {
```

Comment:
Just name it `StatisticsComponent`

Reply:
[Done](https://github.com/ls1intum/Artemis/pull/2471/commits/499bb9fd70d8f4710fe3744189d0493f61a8db7d)